Tasks/distinct.js

```diff
@@ -1,19 +1,15 @@
 // Return an array without duplicates
 
-DISTINCT = (data) => {
-  console.log({ data });
-  A = new Set();
-  w = 0;
-  data.forEach((a) => {
-    if (A.has(a)) {
-      delete data[w];
-    } else {
-      A.add(a);
-    }
-    w++;
-  });
-  return data.filter
-  (x => typeof x === 'number');
+"use strict";
+
+const distinct = array => {
```

Comment:
Use eslint before commit, we have a rule to have round braces always, even for single argument function
```suggestion
const distinct = (array) => {
```